### ISSUE TITLE
Phase 2: Move content-build Nightwatch E2E to GHA

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#25309]
# Description
The `content-build` repository does not currently have a GitHub Actions workflow that executes its minimal set of Nightwatch and Cypress test specs. We need to create a workflow or add to the existing `Continuous Integration` workflow so that the Nightwatch and Cypress E2E test specs are executed as part of the `content-build` build process conducted in GitHub Actions.

## Acceptance Criteria
- [ ] The `content-build` GitHub Actions build executes Nightwatch E2E tests
- [ ] Nightwatch and Cypress tests are executed in parallel as part of the same workflow
- [ ] E2E results are reported in aggregate

**Comment by ddzz** [comment]
WIP branch: https://github.com/department-of-veterans-affairs/content-build/compare/nightwatch-e2e-gha?expand=1

**Comment by pjhill** [comment]
This has been merged for a couple weeks. There are currently no issues with this element of the content-build. If any issues arise later then we will work those under a new ticket.
